example_agents/sb3_agent/components.yaml

```diff
@@ -24,7 +24,7 @@ components:
         requirements_path: ./requirements.txt
         dependencies:
             AtariEnv: AtariEnv==db3728264f382402120913d76c4fa0dc320ef59f
-            CartPoleEnv: CartPoleEnv==4ede9280f9c477f1ca09929d10cdc1e1ba1129f1
+            CartPoleEnv: CartPoleEnv==c755d5c35a25ab118746e2ba885894ff66fb8c43:
```

Comment:
Same here. Did you intend for there to be a colon at the end of this line? Seems like these colons might be breaking the sb3 tests (which seem to be failing when this file is parsed).

Reply:
This, however, is a typo!